src/data/complex/basic.lean

```diff
@@ -63,7 +63,11 @@ instance : can_lift ℂ ℝ :=
   coe := coe,
   prf := λ z hz, ⟨z.re, ext rfl hz.symm⟩ }
 
-notation s ` ×ℂ `:72 t := re ⁻¹' s ∩ im ⁻¹' t
+/-- The product of a set on the real axis and a set on the imaginary axis of the complex plane,
+denoted by `s ×ℂ t`. -/
+def _root_.set.re_prod_im (s t : set ℝ) : set ℂ := re ⁻¹' s ∩ im ⁻¹' t
```

Comment:
suggested name: `complex.rect`

Reply:
That's a fair point, but the terminology "rectangle" to refer to products of sets is not that unusual either; for example the basis elements in the product topology are often called rectangles.

Reply:
See https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.2311770.20re-im.20set.20product/near/270496094 for a poll on the naming issue.